### DISCUSSION TITLE
No cron/anacron service?

### DISCUSSION THREAD

**Discussion by tve** [Manjaro-Sway/manjaro-sway#355] [Q&A] [answered]
Looks like cron and anacron are not set-up at all? I'm not seeing any systemd service files either (but could be looking in the wrong place). Is this intentional or an oversight?
Edit: looks like the 'cronie' service file starts cron, but it's not enabled by default.

**Comment by boredland** [comment]
We could add that to our iso profile. Should we? Not sure how other distros do that these days. Haven't used a Cron on a non-server in years.

**Reply by tve** [reply]
You're probably right. I use it for btrfs snapshots. Since it's a simple `systemctl enable --now cronie.service` I wouldn't worry more about it. I believe it's enabled by default in the Cinnamon and KDE editions, but I could be wrong.

**Reply by boredland** [reply]
you're right: https://github.com/manjaro-sway/iso-profiles/blob/sway/manjaro/kde/profile.conf#L43

**Reply by boredland** [reply]
I am not too emotional about it. It should not take a lot to have it enabled.

**Comment by boredland** [comment, marked as answer]
Cronie should be enabled in future isos
